README.md

```diff
@@ -300,8 +300,7 @@ A cuda device is optional and will be auto-detected.
 The following command extracts the final-layer embedding for a FASTA file from the ESM-2 model:
 
 ```bash
-$ python scripts/extract.py esm2_t33_650M_UR50D examples/data/some_proteins.fasta examples/data/some_proteins_emb_esm2/ \
-    --repr_layers 0 32 33 --include mean per_tok
+python scripts/extract.py esm2_t33_650M_UR50D examples/data/some_proteins.fasta examples/data/some_proteins_emb_esm2 --repr_layers 0 32 33 --include mean per_tok
```

Comment:
You can remove the `$` here as well, but the multilines end with `\` so they will work; and I prefer to keep it like that for readability. In this specific case even a newline after the input fasta would improve things.

Reply:
Done!